core/src/test/scala/org/apache/spark/scheduler/EventLoggingListenerSuite.scala

```diff
@@ -251,6 +261,217 @@ class EventLoggingListenerSuite extends SparkFunSuite with LocalSparkContext wit
     }
   }
 
+  /**
+   * Test executor metrics update logging functionality. This checks that a
+   * SparkListenerExecutorMetricsUpdate event is added to the Spark history
+   * log if one of the executor metrics is larger than any previously
+   * recorded value for the metric, per executor per stage. The task metrics
```

Comment:
i think you mean *executor* metrics, not *task* metrics, right?

Reply:
Woops, that was left over from when it was ExecutorMetricsUpdated.